### ISSUE TITLE
Address international field, issue with selecting city

### ISSUE THREAD

**Issue by iamrobert** [FLEXIcontent/flexicontent-cck#346]
The Address 3 and City seem to have the same value.

1. If I write the City in the City Field and click apply - it goes blank.
2. It works if I write it on the 3rd line of the Address

![addressfield](https://cloud.githubusercontent.com/assets/8617673/10932473/fd8f251e-830b-11e5-9fcc-64939d9356bc.png)

I am using FLEXIcontent 3.0.9-rc.

**Comment by ggppdk** [comment]
This sounds like a bug, added milestone v3.0.9

**Comment by ggppdk** [comment]
Wrong field name used will commit fix, 'addr3' was used instead of 'city'
```
$newpost[$new]['city']  = flexicontent_html::dataFilter($v['addr3'],  4000, 'STRING', 0);
```

**Comment by ggppdk** [comment]
Fixed with 5f166af
